scripts/gee

```diff
@@ -2845,26 +2845,63 @@ function gee__pr_make() {
   mapfile -t PARENT_PRS < <( _list_open_pr_numbers "$(_get_parent_branch)" )
   local BODYFILE
   BODYFILE="$(mktemp -t prbody.XXXXXX.txt)"
-  printf "PR generated by %q from branch %q.\n" "${USER}" "${CURRENT_BRANCH}" \
-    >"${BODYFILE}"
-  if (( "${#PARENT_PRS[@]}" > 0 )); then
-    printf "Derived from branch %q's PR #%d.\n" "${PARENT}" "${PARENT_PRS[0]}" \
-      >>"${BODYFILE}"
-  fi
-  printf "\nCommits:\n" >>"${BODYFILE}"
-  "${GIT}" log --format="*  %h %s" --reverse \
-    "${PARENT}...${CURRENT_BRANCH}" >>"${BODYFILE}"
+  (
+    cat <<'EndOfPrTemplate'
+# Pull request template
+#
+# This template is designed to help you write a good PR description.  This
+# description will be used as the first comment for your pull request, and will
+# ultimately become the squash-merge commit message for your change when you
+# run "gee pr_submit".  Note that all lines that start with a "#" will be
+# deleted.
+#
+# The first line if your PR description should be the PR title.  This
+# description should describe what is being affected, and what the change does.
+# For example: "scripts/gee: Add new PR template."
+
+
+# In this section, explain why the PR is necessary and what the PR desires to
+# achieve.
+
+
+# In this section, describe what testing was done to validate your change.
+
+
+# In this section, list any relevant Jira tickets.
```

Comment:
Add an example?

```
# Example:
# Jira: PROJ-1234,PROJ-1235
```

As a broader point, there's prior art for a change footer that contains key-value pairs that allow for easy integration with tooling (Critique does `SCREAMING_SNAKE_CASE_KEY=value`; Gerrit does `Upper-Kebab-Case-Key: value`). Maybe if we pick a strategy and begin to encourage it for these types of fields, it will make future tooling easier?

Reply:
Done.